app/uk/gov/hmrc/fileupload/ApplicationLoader.scala

```diff
@@ -93,6 +95,19 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
     envelopeRepository.update() _
   }
 
+  def envelopeConstraintsConfiguration(config: Configuration) = {
+
+    def throwException(key: String) = {
+      throw new IllegalStateException(s"default value $key need to define")
+    }
+
+    DefaultEnvelopeConstraints(
+                               config.getInt("envelopeDefaultConstraints.maxNumFiles").getOrElse(throwException("maxNumFiles")),
```

Comment:
why not use play-config's get methods in [ServicesConfig](https://github.com/hmrc/play-config/blob/master/src/main/scala/uk/gov/hmrc/play/config/ServicesConfig.scala)?